src/queries/useAuthorities/useAuthorities.js

```diff
@@ -74,7 +74,7 @@ const useAuthorities = ({
   sortedColumn,
 }) => {
   const ky = useOkapiKy();
-  const [namespace] = useNamespace();
+  const [namespace] = useNamespace({ key: 'authoritiesBrowse' });
```

Comment:
Let's rename `authoritiesBrowse` to `authorities` so it's not only browse or search related, but more general name

Reply:
fixed